tests/Sniffs/Namespaces/ImportFixture.php

```diff
@@ -14,9 +14,13 @@
 use const My\Full\MY_CONST;
 // The following line should report an unused import
 use const My\Full\UNUSED_CONST;
+use First;
```

Comment:
This one is unused too, right?

Reply:
Ohhhhh. You're right, because the path to `First` below is absolute.